src/wallet/addresses.go

```diff
@@ -59,7 +59,7 @@ func GetSkycoinWalletEntry(pub cipher.PubKey, sec cipher.SecKey) ReadableEntry {
 	}
 }
 
-// Returns a ReadableEntry in Bitcoin format
+// GetBitcoinWalletEntry returns a ReadableEntry in Bitcoin format
```

Comment:
Does Golint require the function name repeated in the comment string?

Reply:
Yes, every exported function's comment should start with the function name